bridge/engine/EngineMixer.cpp

```diff
@@ -3190,4 +3192,100 @@ void EngineMixer::processRecordingMissingPackets(const uint64_t timestamp)
     }
 }
 
+class SetMaxMediaBitrateJob : public jobmanager::CountedJob
```

Comment:
Please move this to the anonymous namespace in the cpp file, or a separate file.